autotest/pyscripts/test_gdal_calc.py

```diff
@@ -332,6 +332,67 @@ def test_gdal_calc_py_6():
 
     return 'success'
 
+###############################################################################
+# test --optfile
+
+def test_gdal_calc_py_7():
```

Comment:
sorry, this is not the unittest module . You have to manually add test_gal_calc_py_7 to the gdaltest_list array at the bottom of the file